### PR TITLE
Allow 3.2

### PR DESCRIPTION
**Pull request by adamzapasnik**

### CODE REVIEW

**Review by jydli — COMMENTED** [review]
Thanks for contributing to this repo! Just a quick question and a comment on the versioning.

File: ruby_audit.gemspec [hunk 1]
```diff
@@ -17,14 +17,14 @@ Gem::Specification.new do |spec|
...
-  spec.required_ruby_version = ['>= 2.5', '< 3.2']
+  spec.required_ruby_version = ['>= 2.5', '< 3.3']
   spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
   spec.bindir        = 'exe'
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
   spec.add_dependency 'bundler-audit', '~> 0.9.0'
-  spec.add_development_dependency 'pry', '~> 0.13.0'
+  spec.add_development_dependency 'pry', '~> 0.14.1'
```
[the remark this answers is not in the document]
Comment (adamzapasnik) [reply ruby_audit.gemspec:27]: sorry I was being lazy, https://github.com/pry/pry/issues/2261 14.0 introduced a fix to work with Ruby 3.2

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+### Added
```
[the remark this answers is not in the document]
Comment (adamzapasnik) [reply CHANGELOG.md:8]: sure I can do it tomorrow morning (or you can merge and push the version update yourself 🙈 )

[the remark this answers is not in the document]
Comment (adamzapasnik) [reply CHANGELOG.md:10]: amended

**Review by jydli — APPROVED** [review]
LGTM.

Once this PR has been merged, I'll go ahead and cut a new release!

**Comment by xmunoz** [comment]
Who has the ability to merge this PR?

**Comment by jydli** [comment]
> Who has the ability to merge this PR?

Sorry, just merged and cut a new release!

**Comment by xmunoz** [comment]
> Sorry, just merged and cut a new release!

Not sure what your release process is, but the new release does not appear here: https://rubygems.org/gems/ruby_audit

**Comment by jydli** [comment]
> > Sorry, just merged and cut a new release!
> 
> Not sure what your release process is, but the new release does not appear here: https://rubygems.org/gems/ruby_audit

Thank you bringing this up - I am currently in the process of getting the new release pushed and will update this PR once it's been done.

**Comment by jydli** [comment]
> > > Sorry, just merged and cut a new release!
> > 
> > 
> > Not sure what your release process is, but the new release does not appear here: https://rubygems.org/gems/ruby_audit
> 
> Thank you bringing this up - I am currently in the process of getting the new release pushed and will update this PR once it's been done.

The new version should now be available here: https://rubygems.org/gems/ruby_audit

**Pull request merged**